bootCampTopic2/bootCampTopic2ex1/src/test/java/maven/exercise/bootCampTopic2ex1/Exercise1Test.java

```diff
@@ -0,0 +1,75 @@
+package maven.exercise.bootCampTopic2ex1;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+
+
+
+public class Exercise1Test {
+
+
+	@Test
+	public void firstRun() {
+		assertTrue(Exercise1.getRecentFileList().isEmpty());
+		System.out.println("");
+		System.out.println("Test 1: First Run");
+	    Exercise1.showRecentFileList();
+	}
+	
+
+	
+	@Test	
+	public void openFile() throws IOException {
+		 System.out.println("");
+		 System.out.println("Test 2: Add File To Recent File List");	 
+		 String fileOpened;		
+		 File objetofile = new File ("Contacts.txt");		 
```

Comment:
I created this variable to try to create a file and get its name to add to the list